lib/web_ui/lib/src/engine/text/line_break_properties.dart

```diff
@@ -0,0 +1,2771 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// AUTO-GENERATED FILE.
+// Generated by: tool/unicode_sync_script.dart
+//
+// Source:
+// # LineBreak-12.1.0.txt
+// # Date: 2019-03-31, 22:04:15 GMT [KW, LI]
+// # © 2019 Unicode®, Inc.
+// # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+// # For terms of use, see http://www.unicode.org/terms_of_use.html
+
+// @dart = 2.6
+part of engine;
+
+enum LineCharProperty {
+  CM,
+  BA,
+  LF,
```

Comment:
Is it possible to extract docs for these as well?

Reply:
The raw file doesn't contain docs. But the unicode spec web page has a table that explains each one of these. I'll add a link to the table.